x-pack/metricbeat/module/coredns/stats/stats_integration_test.go

```diff
@@ -18,8 +18,8 @@ import (
 func TestFetch(t *testing.T) {
 	service := compose.EnsureUp(t, "coredns")
 
-	f := mbtest.NewReportingMetricSetV2(t, getConfig(service.Host()))
-	events, errs := mbtest.ReportingFetchV2(f)
+	f := mbtest.NewReportingMetricSetV2Error(t, getConfig(service.Host()))
```

Comment:
There is another place here where `mbtest.NewFetcher` could be used :slightly_smiling_face: